src/shipping/src/telemetry_conf.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use log::Level;
+use opentelemetry::global;
+use opentelemetry_appender_log::OpenTelemetryLogBridge;
+
+use opentelemetry_resource_detectors::{OsResourceDetector, ProcessResourceDetector};
+use opentelemetry_sdk::{
+    propagation::TraceContextPropagator, resource::ResourceDetector, Resource,
+};
+
+use std::{env, str::FromStr};
+
+fn get_resource() -> Resource {
```

Comment:
I think we should explicitly add the service.* ones too and not rely on defaults.
 
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-otlp/examples/basic-otlp-http/src/main.rs#L23

Reply:
We are passing OTEL_SERVICE_NAME and OTEL_RESOURCE_ATTRIBUTES as env vars.

We use this approach in all services to avoid hardcoding those values.

Reply:
perfect! Ya I forgot about the env variables.